www/front_src/src/Resources/columns/Graph/index.tsx

```diff
@@ -22,6 +22,9 @@ import HoverChip from '../HoverChip';
 import { getData } from '../../api';
 import { ColumnProps } from '..';
 
+const JSXXAxis = (XAxis as unknown) as (props) => JSX.Element;
+const JSXYAxis = (YAxis as unknown) as (props) => JSX.Element;
```

Comment:
recharts decided to include types, but it now breaks our build :D

I'll open an issue on their repo.